include/git2/signature.h

```diff
@@ -30,8 +30,8 @@ GIT_BEGIN_DECL
  * @param out new signature, in case of error NULL
  * @param name name of the person
  * @param email email of the person
- * @param time time when the action happened
- * @param offset timezone offset in minutes for the time
+ * @param time time (in seconds from epoch) when the action happened
+ * @param offset timezone offset (in minutes) for the time
```

Comment:
Hmm, these aren't _minutes_ exactly, this is minutes and hours in ISO 8601 format.  (An hour is _100_ not _60_.)  Would you mind fixing that up while you're here?

Reply:
Sorry, invoking ISO8601 just made me 😨. You mean that it's represented as a 4-digit `hhmm` number ?

Reply:
As far as I can see, [parsing](https://github.com/libgit2/libgit2/blob/master/src/signature.c#L256) does what you expect, but we wrangle this as a [number](https://github.com/libgit2/libgit2/blob/master/src/signature.c#L314) of [minutes](https://github.com/libgit2/libgit2/blob/master/src/signature.c#L171) internally.

Reply:
Yeah, sorry, `hhmm`.

Reply:
I might be misunderstanding, but it looks like the `hhmm` decimal format only relates to parsing signatures, not the public API. My "minutes" link above uses `difftime` (hence seconds) to calculate the offset in minutes, and there's no `100` anywhere.

Reply:
Ha, yes, of course you're right.